docs/developers/how-to/testing.rst

```diff
@@ -0,0 +1,103 @@
+=============
+Running tests
+=============
+
+MicroOVN has two types of tests, linter checks and functional tests.
+
+Linter checks
+-------------
+
+Linting is currently applied only to the tests themselves, not the main
+codebase. The prerequisites for running linter are:
+
+* ``make``
+* ``shellcheck``
+
+To perform linting, run following command:
+
+.. code-block:: none
+
+   make check-lint
+
+Functional tests
+----------------
+
+These tests build the MicroOVN snap and use it to deploy the ``OVN`` cluster
+in LXD containers. This cluster is then used for running functional test
+suites.
+
+Preparing environment
+~~~~~~~~~~~~~~~~~~~~~
+
+There is no need to run tests in dedicated VMs or in isolated environments as
+all functional tests run inside containers and no changes are made to the host
+running them.
+
+Functional tests need to build MicroOVN snap and run multiple containers via
+LXD, so make sure you first complete steps and requirements for
+:doc:`building MicroOVN from source <building>`.
+
+The final step is to ensure that you have installed required
+`Bash Automated Testing System (BATS)`_. Due to the reliance on its latest
```

Comment:
As in the previous how-to, let's create, as **first** section:

Install required software

*use similar approach to other how-to*

Reply:
As I went with `Build requirements` in the `building.rst`, I'd use `Functional test requirements` here.